packages/edit-post/src/components/header/options-menu-item/index.js

```diff
@@ -3,18 +3,24 @@
  */
 import { withDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
-import { MenuItem } from '@wordpress/components';
+import { MenuItem, MenuGroup } from '@wordpress/components';
 
 export function OptionsMenuItem( { openModal, onSelect } ) {
 	return (
-		<MenuItem
-			onClick={ () => {
-				onSelect();
-				openModal( 'edit-post/options' );
-			} }
-		>
-			{ __( 'Options' ) }
-		</MenuItem>
+		<MenuGroup
+			role={ 'group' }
+			useEventToOffset={ false }
+			label={ __( 'Options' ) }
+			isScreenReaderLabel={ true }>
+			<MenuItem
+				onClick={ () => {
+					onSelect();
+					openModal( 'edit-post/options' );
+				} }
+			>
+				{ __( 'Options' ) }
+			</MenuItem>
+		</MenuGroup>
```

Comment:
I wouldn't change this in a group: it's a single menu item within the main parent group. Also, it doesn't need a label that just repeats the name of the item. I'd just revert all the changes to this file.